run_scripts/Readme.md

```diff
@@ -0,0 +1,40 @@
+## Step 1. Create the Loader and execute scripts. 
```

Comment:
Add a prerequisites steps that says this assumes the following 3 exists.
1. a (large) Yugabyte cluster
2. a cluster of client machines
3. a client-manager machine that can ssh into the client machines. 
This script is assumed to be ran on the latter (i.e. the client-manager machine),

Reply:
Done.